### ISSUE TITLE
insightface module for custom nodes due to the lack of NODE_CLASS_MAPPINGS.

### ISSUE THREAD

**Issue by Rachel-2024** [deepinsight/insightface#2682]
Skip /Users/rechel/Desktop/AI/ComfyUI__portable/ComfyUI/custom_nodes/insightface module for custom nodes due to the lack of NODE_CLASS_MAPPINGS.

Failed to import node, resulting in inability to use

How should I solve it?

**Comment by polarNerd204** [comment]
yeah I have this aswell.....it appears to be the last hurdle in the weeks long battle with insightface on my mac M2. I have the reActor node importing fine now this is hopfully the last thing. Did you find a solution?
